doc/source/release-notes/Percona-Server-5.6.27-76.0.rst

```diff
@@ -0,0 +1,39 @@
+.. rn:: 5.6.27-76.0
+
+==============================
+ |Percona Server| 5.6.27-76.0 
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.27-76.0 on December 2nd, 2015 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.27-76.0/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.27 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-27.html>`_, including all the bug fixes in it, |Percona Server| 5.6.27-76.0 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.27-76.0 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.27-76.0>`_.
+
+New Features
+============
+
+ ``SHOW SLAVE STATUS NOLOCK`` syntax in 5.6 has been undeprecated. Both ``SHOW SLAVE STATUS NOLOCK`` and ``SHOW SLAVE STATUS NONBLOCKING`` are now supported.
+
+ |TokuDB| tables can now be automatically :ref:`analyzed in the background <tokudb_background_analyze_table>` based on a measured change in data. If you're upgrading an existing |TokuDB| installation, the |TokuDB| plugin must be disabled and re-enabled via :program:`ps_tokudb_admin` to register this new plugin. Alternately the plugin can be enabled by running: 
+ 
+   .. code-block:: mysql
+   
+     INSTALL PLUGIN tokudb_background_job_status SONAME 'ha_tokudb.so';
+
+ |TokuDB| ``INFORMATION_SCHEMA`` tables and system variables have been refactored into their own files and namespaces.
+
+ |TokuDB| code has been cleaned up and modernized to facilitate ease of adding new features and maintenance.
+
+ |Percona Server| has implemented new :variable:`tokudb_strip_frm_data` variable which can be used to assist in |TokuDB| data recovery.
```

Comment:
Needs a strong warning to know what one is doing in order to use it. (in feature doc below and here too)